File: findMin11.py
arr = []
# 5
# 9 5 2 7 4

n = input()  # 5
a = input()  # "9 5 2 7 4"
tmp = a.split() # ["9", "5", "2", "7", "4"]

for i in tmp:
    arr.append(int(i))  # int(i)    arr = [9, 5, 2, 7, 4]

min = arr[0]

for i in arr:
    if i< min:
        min = i

print(min)

File: findMin11.cpp
#include <stdio.h>
int main(){

  int arr[100] = {0,};

  int n;    // n <- 불러올 숫자의 갯수 5
  // int 타입은 4개의 바이트를 사용합니다.
  // RAM 램.... 4G ... "0[0 또는 1] ~ 40억"
  int min;
  int i;

  scanf("%d", &n); // 10
                    // %d - 정수, %c - 문자, %f - 실수
                    // 5,7,9,2,4,5,8,9,10,8
  for(int i=0; i<n; i++) {
    scanf("%d", &arr[i]);
  }

  min = arr[0]; // 5

  for (i=0; i<n; i++) {
    if (arr[i]<min){
        min = arr[i];
    }
  }
  printf("%d", min);
  return 0;
}
